SimCalorimetry/HGCalSimProducers/interface/HGCFEElectronics.h

```diff
@@ -34,7 +34,7 @@ class HGCFEElectronics {
                         hgc::HGCSimHitData& chargeColl,
                         hgc::HGCSimHitData& toa,
                         CLHEP::HepRandomEngine* engine,
-                        int thrADC = -1,
+                        uint32_t thrADC = -1,
```

Comment:
This is effectively setting the default value to the highest possible uint32_t. Is this desirable?

Reply:
@deguio can you comment on this?

Reply:
thanks @kpedro88 for catching this. this is now fixed and initialized to 0 in case a value is not explicitly passed.